app/poi_api/poi_suggestion.py

```diff
@@ -0,0 +1,28 @@
+from datetime import datetime
+
+
+class POI_suggestions:

```

Comment:
Is this meant to be a data class? If so, it should be inheriting from FirebaseDataEntity